### PR TITLE
Add Slovene and Turkish to languages.json / Fix CI

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
This PR adds Slovene and Turkish ("sl" and "tr") to languages.json. ~~The changes here were generated by the [localization tool](https://github.com/mozilla-mobile/mozilla-vpn-client/tree/main/tools/languagelocalizer), which required a bit of duct taping before it would Just Work™. Ergo, I am suspicious of some of the deletions, particularly those around L81.~~

This should unblock the functional tests. See[ this PR](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/6867) for an excellent summary of why.

## Reference

VPN-4874, VPN-4875
## Checklist
    
- [ ] My code follows the style guidelines for this project
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

**Comment by flodolo** [comment]
> which required a bit of duct taping before it would Just Work™

It would help to explain that the changes were, because this doesn't look right.

It's losing an entire language (Corsican), and losing several language names across different locales, but not everywhere (interestingly enough, all of them with underscores, like es_MX)

File: src/apps/vpn/translations/extras/languages.json [hunk 1]
```diff
@@ -78,32 +78,7 @@
       "XXX": "¤"
     },
     "languages": {
-      "bg": "Lingua bulgara",
```
Comment (bakulf) [inline src/apps/vpn/translations/extras/languages.json:81]: This part is suspicious... I wonder why so many lines deleted.

**Review by bakulf — CHANGES_REQUESTED** [review]
Usually what I did was to reintroduce deleted lines. No particular reasons, but just to have more language coverage.
The issue is that Wikipedia changes really often and there is not a good review process for these language details.

File: src/apps/vpn/translations/extras/languages.json (hunk 1 above)
Comment (lesleyjanenorton) [reply src/apps/vpn/translations/extras/languages.json]: Agreed. Reverted these changes.

**Comment by lesleyjanenorton** [comment]
@flodolo Deletions reverted. I'm not sure if they were caused by some hacking I did to get the script running locally and/or changes to wikidata or the unicode [CLDR Project repository](https://github.com/unicode-org/cldr-json). Agree about the documentation and filed VPN-4875 for clarification.

**Review by bakulf — APPROVED** [review]

**Pull request merged**